kytos/core/interface.py

```diff
@@ -494,7 +494,7 @@ def __eq__(self, other):
 
     def is_valid(self):
         """Check if TAG is possible for this interface TAG pool."""
-        if self.user_tag:
+        if self.user_tag and isinstance(self.user_tag.value, int):
```

Comment:
@Alopalao in the core and its validation with `is_valid` we'd like to only accept values that indeed are part of the tag type domain to minimize the surface of potential bugs. If we were to ignore any strings then it could be instantiated with whatever. Thinking about vlan range as well, and how `dl_vlan` PRs are evolving on `flow_manager`, we need to make sure that if a string of a range is passed then the range also needs to be validated (this will be in a future PR, but we don't want to leave this too open). 

Could you first please finish the issue discussions and determine wich value/mask will be represented by untagged and any and sync with @ajoaoff with the proposal that you guys would need? We can accept `"any"` or `"untagged"` on the `tag.value` since they won't need to reserve a vlan tag from the pool and only keep numbers in the `available_tags`, but let's also validate that only the strings that are part of our domain is accepted and make sure that's sufficient for what we need. Thanks, Aldo.

We'll also probably need to extend `TAG` to have which set of tags it represents and not only `value` to have an efficient way of checking intersections and unions. mef_eline [`shares_uni`](https://github.com/kytos-ng/mef_eline/blob/master/models/evc.py#L303-L310) will also need to be adapted when trying to check if a new the circuit being created shares uni vlans

Reply:
Got it. I also finished `mef_eline` with the required cases and they all are working now. I am working on its tests and I just pushed this one first since it was the fastest to do.

Reply:
Cool. It's shaping up nicely, great progress. 

Aldo, since these two are independent issues but somewhat related, please feel free to take the initiative documenting what you want to propose and what you have so far on https://github.com/kytos-ng/mef_eline/issues/219, and that way for things related to vlan range we can leave in a way where we can extend and easily evolve to it. As Antonio, start evolving vlan range discussion then he can also use the information there and be aware how it's evolving.

Reply:
Reduced the accepted vales on commit [c5294a7e1011040e81d9d5a466df6f53b6dd3369](https://github.com/kytos-ng/kytos/pull/331/commits/c5294a7e1011040e81d9d5a466df6f53b6dd3369)

Reply:
Looks good. Let me close this thread.